app/workers/send_chase_email_to_referees_worker.rb

```diff
@@ -0,0 +1,13 @@
+class SendChaseEmailToRefereesWorker
+  include Sidekiq::Worker
+
+  def perform
+    GetRefereesToChase.new.perform.each do |reference|
+      begin
+        SendChaseEmail.new.perform(refence: reference)
```

Comment:
Just realised this needs to be more explicit as there will be multiple chase emails
Perhaps SendRefereeChaseEmail?
Or SendFirstChaseEmailToReferee